cirq-google/cirq_google/engine/engine.py

```diff
@@ -210,21 +210,19 @@ def __init__(
     def __str__(self) -> str:
         return f'Engine(project_id={self.project_id!r})'
 
-    # TODO(#6271): Deprecate and remove processor_ids before v1.4
     def run(
         self,
         program: cirq.AbstractCircuit,
         program_id: Optional[str] = None,
         job_id: Optional[str] = None,
         param_resolver: cirq.ParamResolver = cirq.ParamResolver({}),
         repetitions: int = 1,
-        processor_ids: Sequence[str] = ('xmonsim',),
+        processor_id: str = "xmonsim",
```

Comment:
@wcourtney should I also remove the default and make it required here?

Also, should I remove the `ValueError: If 'processor_id' is not set` comment?